packages/react/src/select/root/SelectRoot.tsx

```diff
@@ -69,7 +69,7 @@ const SelectRoot: SelectRoot = function SelectRoot<Value>(
         <input
           {...rootContext.fieldControlValidation.getInputValidationProps({
             onFocus() {
-              // Move focus from the hidden <select> to the trigger element.
+              // Move focus to the trigger element when the hidden input is focused.
               rootContext.triggerElement?.focus();
```

Comment:
Small amendment: unlike Checkbox and Switch, with Select it's still pointing to the hidden `<input>`, but focused is moved straight away to the trigger. This prevents the warning and also prevents the select from opening when clicking on the label unexpectedly.